### PR TITLE
docs: fix the documentation of the balance

### PR DESCRIPTION
**Pull request by vrolland**

### CODE REVIEW

File: packages/docs/docs/guides/3-Portal-API/2-payment-status.md [hunk 1]
```diff
@@ -3,52 +3,59 @@ title: Payment status with the Portal API
...
+You can use the following snippet to see if the request has been paid.
 
 ```jsx
 // Import Big Number package
-const BN = require('bn.js')
-
+const BN = require('bn.js');
 (async () => {
   // Check the balance of the request
   const requestData = request.getData();
```
Comment (benjlevesque) [inline packages/docs/docs/guides/3-Portal-API/2-payment-status.md:41]: this line should go away. Reusing your example from above:
```
const request = result.data.request;
const balanceObject = requestData.balance;

```

File: packages/docs/docs/guides/3-Portal-API/2-payment-status.md [hunk 2]
```diff
@@ -3,52 +3,59 @@ title: Payment status with the Portal API
...
-  const balance = requestData.balance;
-  console.log(`Balance of the request in ETH: ${balance}`);
-  
+  const balanceObject = requestData.balance;
+
+  if (balanceObject?.error) {
+    console.error(balanceObject.error.message);
+    return;
```
Comment (benjlevesque) [inline packages/docs/docs/guides/3-Portal-API/2-payment-status.md:46]: 2 comments here:  we should handle case when balanceObject is not set insteady of using `!`, and I'm not sure there's a `message` field on `balanceObject.error`... isn't it plain text?

```suggestion
  if (!balanceObject){
    console.error("balance not set");
    return;
  }
  if(balanceObject.error) {
    console.error(balanceObject.error);
    return;
```

File: packages/docs/docs/guides/3-Portal-API/2-payment-status.md [hunk 3]
```diff
@@ -3,52 +3,59 @@ title: Payment status with the Portal API
...
+  }
+
+  console.log(`Balance of the request in ETH: ${balanceObject?.balance}`);
```
Comment (benjlevesque) [inline packages/docs/docs/guides/3-Portal-API/2-payment-status.md:49]:
```suggestion
  console.log(`Balance of the request in ETH: ${balanceObject.balance}`);
```

File: packages/docs/docs/guides/3-Portal-API/2-payment-status.md [hunk 4]
```diff
@@ -3,52 +3,59 @@ title: Payment status with the Portal API
...
+
   // Check if the request has been paid
   // Convert the balance to big number type for comparison
   const expectedAmount = new BN(requestData.expectedAmount);
-  const balanceBigNumber = new BN(balance);
+  const balanceBigNumber = new BN(balanceObject?.balance);
```
Comment (benjlevesque) [inline packages/docs/docs/guides/3-Portal-API/2-payment-status.md:54]:
```suggestion
  const balanceBigNumber = new BN(balanceObject.balance);
```

File: packages/docs/docs/guides/3-Portal-API/2-payment-status.md (hunk 2 above)
Comment (vrolland) [reply packages/docs/docs/guides/3-Portal-API/2-payment-status.md:46]:
```
/** Interface for error encounter when getting the balance */
export interface IBalanceError {
  message: string;
  code: BALANCE_ERROR_CODE;
}
```

Comment (benjlevesque) [reply packages/docs/docs/guides/3-Portal-API/2-payment-status.md:46]: ok ! my bad

File: packages/docs/docs/guides/3-Portal-API/2-payment-status.md [hunk 5]
```diff
@@ -3,52 +3,65 @@ title: Payment status with the Portal API
...
 (async () => {
   // Check the balance of the request
-  const requestData = request.getData();
-  const balance = requestData.balance;
-  console.log(`Balance of the request in ETH: ${balance}`);
-  
+  const result = await axios.get(`https://api.request.network/requests/${requestId}`);
+  const request = result.data.request;
+
+  const balanceObject = requestData.balance;
```
Comment (benjlevesque) [inline packages/docs/docs/guides/3-Portal-API/2-payment-status.md:44]:
```suggestion
  const balanceObject = request.balance;
```

**Pull request merged**